Maja/script.js

```diff
@@ -0,0 +1,90 @@
+/* 1. 
+2D Array
+Given the 2D Array below, loop through the arrays to print the values. */
+
+let board = [
+  [1, 2, 3],
+  ["quick", "brown", "fox", "jumped", "over", "lazy", "dog"],
+  [true, false]
+];
+for (let i = 0; i <= board.length; i++) {
+    console.log(board[i]);
+} 
+// [ 1, 2, 3 ]
+// [
+//     'quick', 'brown',
+//     'fox',   'jumped',
+//     'over',  'lazy',
+//   'dog'
+// ]
+// [ true, false ]
+
+/* 2. Doggo
+2.1 Create a doggo object. Add name and breed as properties of the object.
+2.2 Create an array within the doggo object named favorite foods and add the doggo's favorite foods to the array.
+2.3 Access the second element of the doggo's favorite foods.
+2.4 Add a method that loops through and print all the doggo's favorite food. */
+
+let doggo = {
+    name: "Pluto",
+    breed: "Terrier",
+    favoriteFoods: [
+        "bones",
+        "meet", 
+        "eggs"
+    ],
+    looping: function () {
+        for (let i = 0; i <= doggo.favoriteFoods.length; i++) {
```

Comment:
Here again the index iteration, you have the done the same as the previous exercise.